### PR TITLE
Fix url_extract_path Presto function

### PR DESCRIPTION
**Pull request by mbasmanova**
This function needs to unescape the path after extraction.

### CODE REVIEW

File: velox/functions/prestosql/tests/URLFunctionsTest.cpp [hunk 1]
```diff
@@ -16,27 +16,26 @@
 #include <gmock/gmock.h>
 #include "velox/functions/prestosql/tests/utils/FunctionBaseTest.h"
 
-using string_t = std::string;
```
Comment (spershin) [inline velox/functions/prestosql/tests/URLFunctionsTest.cpp:19]: Good riddance. :)

**Review by kgpai — APPROVED** [review]

File: velox/functions/prestosql/tests/URLFunctionsTest.cpp [hunk 2]
```diff
@@ -101,6 +100,17 @@ TEST_F(URLFunctionsTest, validateURL) {
...
+TEST_F(URLFunctionsTest, extractPath) {
+  const auto extractPath = [&](const std::optional<std::string>& url) {
+    return evaluateOnce<std::string>("url_extract_path(c0)", url);
+  };
+
+  ASSERT_EQ(
+      "/media/set/Books and Magazines.php",
+      extractPath(
+          "https://www.cnn.com/media/set/Books%20and%20Magazines.php?foo=bar"));
+}
```
Comment (spershin) [inline velox/functions/prestosql/tests/URLFunctionsTest.cpp:112]: Why do we need a function object?
Why not simply:
```
ASSERT_EQ(
      "/media/set/Books and Magazines.php",
      evaluateOnce<std::string>("url_extract_path(c0)",
          "https://www.cnn.com/media/set/Books%20and%20Magazines.php?foo=bar");
```
?

**Review by spershin — COMMENTED** [review]
Thanks for fixing this!

File: velox/functions/prestosql/tests/URLFunctionsTest.cpp (hunk 2 above)
Comment (mbasmanova) [reply velox/functions/prestosql/tests/URLFunctionsTest.cpp:112]: To make it easy to add more test cases.

Comment (spershin) [reply velox/functions/prestosql/tests/URLFunctionsTest.cpp:112]: Just copy/paste  the ASSERT statement?
The more plain the code is - the easier it is to understand it.

**Pull request closed without merging**